content/post/0-to-60-analytics-stack.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Building an Analytics Stack in 2020"
+author: "Nelson Auner"
+# I can't paint like y'all do!
+# cover: "/img/analysts_or_self_serve.png"
+tags: ["Analytics"]
+date: 2020-03-30T12:16:09-05:00
+draft: false
+---
+
+Congratulations, you are launching an analytics/data function! Maybe you are the first
+analytics hire at a brand new company, or perhaps you are bringing analytics to a mature
+company that historically lacked a comprehensive data strategy. You sold the vision and also put together a roadmap based on your experience and current best practices. What could go wrong (and who could argue that refactoring from a star to snowflake schema is an important company-wide goal)?
+
```

Comment:
Insert `<!--more-->`  after the first paragraph it indicate where the intro ends (the text before `<!--more-->` will appear on the homepage.